app/design/frontend/Magento/blank/web/css/source/_layout.less

```diff
@@ -93,7 +93,14 @@
     }
 
     .page-main {
+        .lib-vendor-prefix-flex-grow(1);
+        .lib-vendor-prefix-flex-shrink(0);
+        .lib-vendor-prefix-flex-basis(auto);
         width: 100%;
+
+        .ie9 & {
+            width: auto;
+        }
```

Comment:
I believe we don't need to support ie9, so this should be removed.

Reply:
Yeah indeed, Magento dropped support for Internet Explorer completely with the release of Magento 2.4.0

Docs: [2.3.x browser support](https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements.html#supported-browsers) vs [2.4.x browser support](https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements.html#supported-browsers)

Reply:
I have removed that